src/guides/v2.3/frontend-dev-guide/templates/template-email.md

```diff
@@ -97,7 +97,9 @@ Some email clients (for example, Gmail) support only CSS styles that have been a
 
 The `<Magento_Email_module_dir>/view/frontend/email/header.html` file contains an `inlinecss` directive:
 
-    {% raw %}{{inlinecss file="css/email-inline.css"}}{% endraw %}
+```html
```

Comment:
the block still needs to be enclosed within {% raw %} {% endraw %} tags to make Liquid parser to ignore the special symbols {{ }}

Reply:
fixed